juri/exercises/exercises.js

```diff
@@ -3,8 +3,17 @@
  * @param {string} str String to verify
  * @returns {boolean} True if "str" is a palindrome
  */
+
+
 export function isPalindrome(str) {
```

Comment:
If you run `npm run test` locally you will see 3 failed:

Palindrome Suite
    ✕ should return false for empty word (3 ms)
    ✕ should return false for null and undefined (1 ms)
  Reverse Suite
    ✕ should return empty string for null and undefined (1 ms)